bootstrapvz/providers/ec2/tasks/ami.py

```diff
@@ -125,3 +125,12 @@ def run(cls, info):
             registration_params['sriov_net_support'] = 'simple'
 
         info._ec2['image'] = info._ec2['connection'].register_image(**registration_params)
+
+        # Setting up tags on the AMI
+        if 'tags' in info.manifest.data and len(info.manifest.data['tags']) > 0:
+            def apply_format(v):
```

Comment:
No worries :-)
What I mean is that you added an entire function (`apply_format`) so you could write:
`tags = {k: apply_format(v) for k, v in raw_tags.items()}`  
instead of  
`tags = {k: v.format(**info.manifest_vars) for k, v in raw_tags.items()}`

It doesn't save you any lines and you add a layer of indirection by doing this.